### PR TITLE
updating based on kudos spec

### PR DESCRIPTION
**Pull request by starpause**

### CODE REVIEW

File: TEMPLATE-SPEC.md [hunk 1]
```diff
@@ -9,69 +9,123 @@ Requires: spec-xyz | ip-xyz
 
 # Spec-<Number>: <Title>
 
-## Summary
+## Overview
```
Comment (robert-zaremba) [inline TEMPLATE-SPEC.md:12]: Let's call it Summary or Abstract - this is a standard for the documents.

File: TEMPLATE-SPEC.md [hunk 2]
```diff
@@ -9,69 +9,123 @@ Requires: spec-xyz | ip-xyz
...
 
-Background and short summary in layman terms.
+<!-- One page high-level overview; put details in the specification section and background in the previous section. Should be understandable by a new engineer or reader who isn't already working on the project. -->
 
-## Context
+Short summary in layman terms.
```
Comment (robert-zaremba) [inline TEMPLATE-SPEC.md:16]: let's be consistent with putting the section description in normal text or in the comment.

File: TEMPLATE-SPEC.md [hunk 3]
```diff
@@ -9,69 +9,123 @@ Requires: spec-xyz | ip-xyz
...
+
+### Challenge
+
+<!-- List the challenge(s) being solved by this widget. What is the problem? -->
+<!-- One page high-level overview; put details in the specification section and background in the previous section. Should be understandable by a new engineer or reader who isn't already working on the project. -->
+
+### Context / Background
```
Comment (robert-zaremba) [inline TEMPLATE-SPEC.md:23]: Let's decide if we want to call it Context or Background. Personally I prefer Context.

File: TEMPLATE-SPEC.md [hunk 4]
```diff
@@ -9,69 +9,123 @@ Requires: spec-xyz | ip-xyz
...
 
 What's the "historical" context? Why do we need a new project?
 Identify the problems and the needs.
 
-## Scope and Requirements
+## Use Cases
+
+<!-- Who is using this and what does it do? Leave "how" to the design/tech specification sections -->
+
+### Actors / Actions
```
Comment (robert-zaremba) [inline TEMPLATE-SPEC.md:34]:
```suggestion
### Actors / Actions
```
```suggestion
### Actors and Actions
```

File: TEMPLATE-SPEC.md (hunk 2 above)
Comment (starpause) [reply TEMPLATE-SPEC.md:16]: OK, I prefer it as a comment.

File: TEMPLATE-SPEC.md (hunk 1 above)
Comment (starpause) [reply TEMPLATE-SPEC.md:12]: Let's keep Overview. Summary is short and part of the overview. Abstract is the opposite of a spec, it's something lacking definition.

File: TEMPLATE-SPEC.md (hunk 3 above)
Comment (starpause) [reply TEMPLATE-SPEC.md]: Context works for me!

**Review by starpause — COMMENTED** [review]
LGTM

File: TEMPLATE-SPEC.md (hunk 1 above)
Comment (robert-zaremba) [reply TEMPLATE-SPEC.md:12]: * NEP templates requires "Summary": https://github.com/near/NEPs/blob/master/nep-0000-template.md
* W3C is also using Summary
* Latest EIPs are using "Abstract": https://github.com/ethereum/EIPs/blob/master/EIPS/eip-6900.md
* Algorand is using "Abstract": https://github.com/algorandfoundation/ARCs/blob/main/ARC-template.md

TBH, I don't recall any design process which uses "overview"

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Let's don't use Overview (detailed in the comment). Accepted options (let's chose one):
* Summary (my preference - NEP is using it)
* Abstract
